docker/scvi/scripts/README.md

```diff
@@ -1,38 +1,41 @@
+# scVI analysis Pipeline python scripts
 
 
-#### _PREPROCESS_
-- _pre-preprocessing_: `cellbender` ('cellbender.py') ~~or `scAR`('preprocess_scar.py').~~  (replacing `soupx` in preprocess.R)
-- _preprocessing_: `scrublet` ('preprocess.py') ~~or`SOLO` ('preprocess_solo.py')~~ 
+### _PREPROCESS_
+- _pre-preprocessing_: [`cellbender`](../../cellbender/scripts/cellbender.py)
```

Comment:
Cellbender is called directly, not with script anymore

Reply:
Right.  I will update the documentation noting that.